### PR TITLE
Migration/staveline

### PR DESCRIPTION
**Pull request by rvilarl**
Staveline!

### CODE REVIEW

File: src/staveline.ts [hunk 1]
```diff
@@ -10,13 +10,37 @@
...
+import { StaveNote } from './stavenote';
+
+export interface StaveLineNotes {
+  last_indices: number[];
+  first_indices: number[];
+  last_note: StaveNote;
+  first_note: StaveNote;
+}
+
+export interface StaveLineRenderOptions {
```
Comment (0xfe) [inline src/staveline.ts:23]: This can be inline.

Comment (rvilarl) [reply src/staveline.ts:23]: done

**Comment by rvilarl** [comment]
Ready to merge!

**Comment by 0xfe** [comment]
Looks good.

**Pull request merged**